### PR TITLE
Remove PHP mailer rule + add rule exception

### PR DESCRIPTION
**Pull request by jkamdjou**

### CODE REVIEW

File: detection-rules/impersonation_github.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source: |
       or ilike(sender.email.email, '*github*')
       or iedit_distance(sender.email.domain.sld, 'github') <= 2
   )
-  and sender.email.domain.root_domain not in ('github.com', 'gitlab.com', 'itthub.net')
+  and sender.email.domain.root_domain not in ('github.com', 'gitlab.com', 'itthub.net', 'githubsupport.com')
```
Comment (rw-access) [inline detection-rules/impersonation_github.yml:14]: how confident are you in this? I don't get any results from a domain lookup

Comment (rw-access) [reply detection-rules/impersonation_github.yml:14]: hmmm, looks like it has MX and TXT records though?
https://www.whatsmydns.net/dns-lookup?query=githubsupport.com&server=google

Comment (jkamdjou) [reply detection-rules/impersonation_github.yml:14]: i've communicated with them before

image

Comment (rw-access) [reply detection-rules/impersonation_github.yml:14]: okay, it's zendesk
https://webcache.googleusercontent.com/search?q=cache:QeIMJ5eQ5BIJ:https://github.community/t/emails-from-support-githubsupport-com/171928+&cd=1&hl=en&ct=clnk&gl=us

https://news.ycombinator.com/item?id=23703453

**Review by rw-access — APPROVED** [review]

**Pull request merged**